1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,77 @@
-// Write your code here
+let images = [{
+        src: "https://static01.nyt.com/images/2021/10/26/science/20tb-horses/20tb-horses-articleLarge.jpg?quality=75&auto=webp&disable=upscale",
+        alt: "herd of horses",
+    },
+    {
+        src: "https://static01.nyt.com/images/2021/03/21/fashion/19EQUUS-1/19EQUUS-1-articleLarge-v4.jpg?quality=75&auto=webp&disable=upscale",
+        alt: "a few horses in barn",
```

Comment:
Good to see you including alt text, but make sure that they're up to date and descriptive